index.bs

```diff
@@ -527,8 +527,9 @@ When this method is invoked, the user agent MUST execute the following algorithm
     |extension| → |argument| of <code>|options|.{{MakeCredentialOptions/extensions}}</code>:
     1. If |extension| is not supported by this client platform, then [=continue=].
 
-    1. Otherwise, let |result| be the result of running |extension|'s [=client processing=] algorithm on |argument|. If the
-        algorithm returned an error, [=continue=].
+    1. Otherwise, let |result| be the result of running |extension|'s [=client extension processing=] algorithm on |argument|. If the
+        algorithm returned an error, return [=a promise rejected with=] either that error or a
```

Comment:
also, this is still fundamentally changing how extensions returning errors are handled, see https://github.com/w3c/webauthn/pull/389#discussion_r107807105 and https://github.com/w3c/webauthn/pull/389#discussion_r109304292.

Is there some way to note the error and [=continue=] with remaining extensions?